src/Console/User.php

```diff
@@ -262,10 +263,15 @@ private function addUser()
 			$lang = CliPrompt::prompt();
 		}
 
+		if (empty($avatar)) {
+			$this->out($this->l10n->t('Enter URL of an image to use as avatar (optional): '));
```

Comment:
Why not take the default avatar URL from the node as avatar if no avatar URL was supplied here? `$base_url/images/person-175.jpg` should be available.

Reply:
We don't need to store the default avatar path in the user record, we should default to it if the avatar isn't set.